src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,211 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    public MyLinkedList() {
+        first = null;
+        last = null;
+        size = 0;
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (size == 0) {
+            first = last = new Node<>(null, value, null);
+            size++;
+            return true;
```

Comment:
this code is duplicated
hint: we can use if-else construction

Reply:
done